jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -126,7 +126,7 @@
     </div>
 
     <div v-if="subset_items.length > 0">
-      <span class="export-category">Spatial Subsets</span>
+      <span class="export-category">Subsets</span>
       <v-row>
         <span class="category-content v-messages v-messages__message text--secondary">Export spatial subset as astropy region.</span>

```

Comment:
this description definitely needs generalizing now (while still keeping this information since that was explicitly requested).